monkey/common/common_consts/environment_variables.py

```diff
@@ -0,0 +1 @@
+AGENT_OTP_ENVIRONMENT_VARIABLE = "IM_OTP"
```

Comment:
This doesn't need to go in common. The Island doesn't need access to it.

On another branch, I'm using `INFECTION_MONKEY_AGENT_OTP`. It's more specific, and there's less chance of a collision. Perhaps there's a rare case where someone already set `IM_OTP` for some unrelated purpose. Unlikely, but possible. Using the longer environment variable: I mean, sure, anything is possible, but very unlikely.

Reply:
Using a longer environment variable name makes the command longer. Could that end up being a problem?
@VakarisZ WDYT?

Reply:
I'm wrong about Island access. My bad there.